meltano/edk/process.py

```diff
@@ -75,14 +76,16 @@ def run(
         to override these you're likely better served using `subprocess.run` directly.
 
         Lastly note that this method is blocking AND `subprocess.run` is called with
-        `check=True`. This means that if the subprocess fails a `CalledProcessError`
-        will be raised.
+        `check=True` and `raise_error=True`. This means that if the subprocess
+        fails a `CalledProcessError` will be raised.
```

Comment:
Can you confirm if `raise_error` isn't doing anything at the moment?

Perhaps the same argument name `check` should be used for this method and just pass it directly to `subprocess.run(..., check=check)`.